manifests/09-prometheus-rules-registry-operations.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: image-registry-rules
+  namespace: openshift-image-registry
+spec:
+  groups:
+  - name: imageregistry.operations.rules
+    rules:
+    - expr: |
+        label_replace(
+          label_replace(
+            sum by (operation) (imageregistry_request_duration_seconds_count{operation="BlobStore.ServeBlob"}), "operation", "get", "operation", "(.+)"
+          ), "resource_type", "blob", "resource_type", ""
+        )
```

Comment:
this feels kinda ugly but it's the only way I found of adding a new label (the outermost `label_replace`) and changing the value of a label (the innermost `label_replace`)

Reply:
Why do we need to rename the labels ? If it is necessary then why not use a simple label_replace:

```
label_replace(
     sum by (operation) (imageregistry_request_duration_seconds_count{operation="ManifestService.Put"}), "operation", "manifestCreate", "operation", "(.+)"
)
```

Reply:
I found that separating resource from operation (i.e manifest / get) improved the overall metric readability.